tests/Index/SettingsTest.php

```diff
@@ -30,11 +30,61 @@ public function testGet(): void
         $this->assertIsArray($settings->get());
         $this->assertNotNull($settings->get('number_of_replicas'));
         $this->assertNotNull($settings->get('number_of_shards'));
+        $this->assertNull($settings->get('max_result_window'));
         $this->assertNull($settings->get('kjqwerjlqwer'));
 
         $index->delete();
     }
 
+    /**
+     * @group functional
+     */
+    public function testGetWithDefaultValueIncluded(): void
+    {
+        $indexName = 'elasticatest';
+
+        $client = $this->_getClient();
+        $index = $client->getIndex($indexName);
+        $index->create([], [
+            'recreate' => true,
+        ]);
+        $index->refresh();
+        $settings = $index->getSettings();
+
+        $this->assertIsArray($settings->get());
+        $this->assertEquals($settings->get('max_result_window', true), 10000);
```

Comment:
```suggestion
        $this->assertEquals(10000, $settings->get('max_result_window', true));
```

Order for `assertEquals` is expected then actual ;)

Reply:
My bad, fixed